UnityProject/Assets/Scripts/Util/ConverterExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static Vector3 To3(this Vector2 other)
 
 	public static Vector3 ToNonInt3(this Vector3Int other)
 	{
-		return new Vector3(other.x, other.y, 0);
+		return new Vector3(other.x, other.y, other.z);
```

Comment:
This might break some stuff since it was Using it as a safeguard?

Reply:
The other thing was something in mobs, not sure why that would need it to be 0

Reply:
basically just afraid of breaking something, I don't mind either way too much